src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -277,6 +286,26 @@ public function get_context( ?string $store_namespace = null ): array {
 			? $context[ $store_namespace ]
 			: array();
 	}
+	/**
+	 * Returns an array representation of the current element being processed.
+	 *
+	 * The returned array contains a copy of the element attributes.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return array|null Current element.
```

Comment:
Ditto above:

```
* @return array{attributes: array<string, string|bool>}|null Current element.
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/7474